### PR TITLE
Fix and unquarantine TestAllDatatypesFromHiveConnector test

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

**Comment by sopel39** [comment]
it it passes, lgtm

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java [hunk 1]
```diff
@@ -86,11 +86,14 @@ private static HiveTableDefinition allHiveSimpleTypesParquetTableDefinition()
                         "   c_bigint             BIGINT," +
                         "   c_float              FLOAT," +
                         "   c_double             DOUBLE," +
+                        "   c_decimal            DECIMAL," +
+                        "   c_decimal_w_params   DECIMAL(10,5)," +
                         "   c_timestamp          TIMESTAMP," +
                         "   c_string             STRING," +
                         "   c_varchar            VARCHAR(10)," +
                         "   c_char               CHAR(10)," +
```
Comment (cawallin) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: at least on master, CHAR isn't a supported datatype. ditto for the above table definition

Comment (losipiuk) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: This is hive definition. 
Assertions do not expect this column to be read by presto.

Comment (losipiuk) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: When Presto CHAR support will be added this test will have to be altered.

Comment (cawallin) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: I know, but there are also quarantined INSERT INTO tests that won't work unless the char column is removed.

**Comment by kokosing** [comment]
lgtm

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java (hunk 1 above)
Comment (losipiuk) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: Oh - that is good point. I did not check if that table definitions are used in some other test. Thanks.

Comment (losipiuk) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: @cawallin: I will leave the `INSERT` test quarantined for now as you are playing with that as part of `FLOAT`. Probably we should (at least for now) split table definitions for `SELECT` and `INSERT` tests as both flows support different subset of types.

**Comment by losipiuk** [comment]
@martint please merge this in while you have a chance :)

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**